### PR TITLE
feat: add new assets (payments)

### LINKED ISSUE #95: feat: add new assets (payments)
Hi,

There are new assets (payments) to add:
- [ ] https://github.com/Decathlon/vitamin-design/issues/110
- [ ] https://github.com/Decathlon/vitamin-design/issues/111
- [ ] https://github.com/Decathlon/vitamin-design/issues/112

Thank you :)

### PR DESCRIPTION
**Pull request by florentlotthepro**
## Changes description

- Close #95 

## Context

There are new assets (payments) to add:
- [x] https://github.com/Decathlon/vitamin-design/issues/110 ➡️ added ✅
- [x] https://github.com/Decathlon/vitamin-design/issues/111 ➡️ added ✅
- [x] https://github.com/Decathlon/vitamin-design/issues/112 ➡️ added ✅

## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [x] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [x] Check your code additions will fail neither code linting checks.
- [x] I have reviewed the submitted code.
- [x] I have tested on an iPhone device/simulator.
- [x] I have tested on an iPad device/simulator.

## Does this introduce a breaking change?

- No

## Screenshots

new_assets_payments

### CODE REVIEW

**Review by daniel-dumortier — APPROVED** [review]
👍
I just merged main into your branch (because I merged another PR into main) before approving to avoid dismissal of my approval when you will try to merge ;)

**Comment by florentlotthepro** [comment]
Thanks 👍
A rebase from the GitHub interface doesn't dismiss existing approvals 😉

**Pull request merged**
